build.gradle

```diff
@@ -1,27 +1,38 @@
-/*
- * This file was generated by the Gradle 'init' task.
- *
- * This generated file contains a sample Java library project to get you started.
- * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
- * User Manual available at https://docs.gradle.org/6.7.1/userguide/building_java_projects.html
- */
-
-
 plugins {
     // Apply the java-library plugin for API and implementation separation.
     id 'java-library'
     id 'maven-publish'
+    id 'se.patrikerdes.use-latest-versions' version '0.2.7'
+
+    id 'checkstyle'
+    id 'jacoco'
+    id 'io.spring.dependency-management' version '1.1.0'
+    id 'com.github.ben-manes.versions' version '0.44.0'
+    id 'org.owasp.dependencycheck' version '8.0.2'
+    id 'com.github.kt3k.coveralls' version '2.8.2'
+    id 'org.sonarqube' version '3.4.0.2513'
+    id "com.jfrog.bintray" version "1.8.5" apply false
+    id "info.solidsoft.pitest" version '1.5.1' apply(false)
+    id "org.jetbrains.gradle.plugin.idea-ext" version "0.7"
+    id 'org.springframework.boot' version '2.7.7'
 }
 
+apply plugin: 'java-library'
 apply plugin: 'java'
```

Comment:
Is the java plugin needed as well as the java-library plugin?

Reply:
removed it

Reply:
Confirmed